<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoUpdateFeatureParameters</class>
 <widget class="QWidget" name="DemoUpdateFeatureParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test transaction</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="PwGisWidget" name="pwgiswidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnOpenMap">
         <property name="text">
          <string>open map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCloseMap">
         <property name="text">
          <string>close map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddPolygone">
         <property name="text">
          <string>add polygon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddStylePolygone">
         <property name="text">
          <string>add style polygone</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btnOpenFile">
         <property name="text">
          <string>open file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReadLine">
         <property name="text">
          <string>read line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtLogLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
